src/minecraft/kotlin/xyz/wagyourtail/unimined/internal/minecraft/patch/fabric/FabricLikeMinecraftTransformer.kt

```diff
@@ -64,7 +65,7 @@ abstract class FabricLikeMinecraftTransformer(
 
     private val include: Configuration = project.configurations.maybeCreate("include".withSourceSet(provider.sourceSet))
 
-    override var customIntermediaries: Boolean by FinalizeOnRead(false)
+    override var customIntermediaries: Boolean by FinalizeOnRead(provider is ReIndevProvider)
```

Comment:
There may be a better way to implement the changes in this file. Instead of checking for `provider is XProvider` there could be a `noMappings` field in `MinecraftConfig`?